### PR TITLE
fix: change Fairs2 artwork grid batch size from 20 to 30

### PR DESCRIPTION
**Pull request by anandaroop**
The type of this PR is: **Bugfix**

This PR resolves [FX-2346]

### Description

The artwork grid on the Fairs2 screen shows artworks in batches of 20. 

This updates the batch size to 30, per the design spec. The new batch size is effect for show more pagination, as well as for refetching in response to filter changes.

Screen Shot 2020-10-12 at 3 42 28 PM

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [ ] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[FX-2346]: https://artsyproduct.atlassian.net/browse/FX-2346

### CODE REVIEW

File: src/lib/Scenes/Fair2/__tests__/Fair2Artworks-tests.tsx [hunk 1]
```diff
@@ -66,6 +67,41 @@ describe("Fair2Artworks", () => {
...
+          ],
+          counts: {
+            total: 30,
+            followedArtists: 0,
+          },
+        },
+      }),
+    })
+
+    expect(wrapper.root.findAllByType(Artwork)).toHaveLength(30)
```
Comment (anandaroop) [inline src/lib/Scenes/Fair2/__tests__/Fair2Artworks-tests.tsx:102]: The mock here does not have 30 items of course, but we couldn't get even a single artwork grid item to render.

File: src/lib/Scenes/Fair2/Components/Fair2Artworks.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { OwnerType } from "@artsy/cohesion"
 import { Fair2Artworks_fair } from "__generated__/Fair2Artworks_fair.graphql"
 import { FilteredArtworkGridZeroState } from "lib/Components/ArtworkGrids/FilteredArtworkGridZeroState"
 import { InfiniteScrollArtworksGridContainer } from "lib/Components/ArtworkGrids/InfiniteScrollArtworksGrid"
+import { FAIR2_ARTWORKS_PAGE_SIZE } from "lib/data/constants"
```
Comment (anandaroop) [inline src/lib/Scenes/Fair2/Components/Fair2Artworks.tsx:5]: We have this pre-existing pattern of declaring this constants elsewhere, so I switched this file over to that.

File: src/lib/Scenes/Fair2/__tests__/Fair2Artworks-tests.tsx [hunk 3]
```diff
@@ -66,6 +66,12 @@ describe("Fair2Artworks", () => {
     expect(wrapper.root.findAllByType(InfiniteScrollArtworksGridContainer)).toHaveLength(1)
   })
 
+  it("requests artworks in batches of 30", () => {
+    const wrapper = getWrapper()
+    const artworksGridContainer = wrapper.root.findByType(InfiniteScrollArtworksGridContainer)
+    expect(artworksGridContainer.props).toMatchObject({ pageSize: 30 })
+  })
```
Comment (anandaroop) [inline src/lib/Scenes/Fair2/__tests__/Fair2Artworks-tests.tsx:73]: Vastly simplified the test approach, no need for all the overeager mocking — just confirm that we are requesting the right page size.

File: src/lib/tests/renderWithWrappers.tsx [hunk 4]
```diff
@@ -38,7 +37,6 @@ export const renderWithWrappers = (component: ReactElement) => {
 
 /**
  * Returns given component wrapped with our page wrappers
- * only <Theme> for now
```
Comment (sweir27) [inline src/lib/tests/renderWithWrappers.tsx:41]: Were these changes intentional?

(I admit the change makes _sense_, but seems slightly unrelated so wanted to make sure!)

Comment (anandaroop) [reply src/lib/tests/renderWithWrappers.tsx:41]: Yep! I was studying up on our test helpers and saw these comments were outdated so I just tweaked em in https://github.com/artsy/eigen/pull/4004/commits/2329dc8ee7e196f2837ab1096a63f9f5edeb7093

**Comment by sweir27** [comment]
Looks good to me!
